### ISSUE TITLE
Use video segmentation annotation for finetune on Charades ?

### ISSUE THREAD

**Issue by v-wewei** [facebookresearch/SlowFast#17]

**Comment by v-wewei** [comment]
Hi, did you use the video segmentation annotation for finetune on Charades dataset ? since, i see you do not mention multi-label loss for  Charades dataset ?

**Comment by takatosp1** [comment]
Hi, 

Thanks for using the codebase, you can find more details about Charade setting in Appendix in https://arxiv.org/pdf/1812.03982.pdf

More specifically:

For Charades, we fine-tune the Kinetics models. A per- class sigmoid output is used to account for the mutli-class nature. We train on a single machine for 24k iterations using a batch size of 16 and a base learning rate of 0.0375 (Kinetics- 400 pre-trained) and 0.02 (Kinetics-600 pre-trained) with 10× step-wise decay if the validation error saturates. For inference, we temporally max-pool scores

**Comment by v-wewei** [comment]
hi, i just get confuse about the sentense "A per- class sigmoid output is used to account for the mutli-class nature", since it do not clearly say what loss function used for training, but only state for the inference. So , i want to know did you train with single label, but test with multi-label. or training with multi-label and test with multi-label .
Thanks vey much !
